src/Routes/Home/Home.jsx

```diff
@@ -0,0 +1,55 @@
+import styles from "./Home.module.css";
+import { SortButton, FilterButton, ProductCard, ToastMsg } from "../../Components";
+import { ProductPage } from "../../Routes";
+import { useProduct, useControl } from "../../hooks";
+
+export const Home = ({ search, isLoading }) => {
+  const {
+    productList,
+    getSortedProductList,
+    getFilteredProductList,
+    getSearchedProductList
+  } = useProduct();
+
+  const { sortBy, filters, isProductSelected } = useControl();
+
+  const sortedProductList = getSortedProductList(productList, sortBy);
+  const filteredProductList = getFilteredProductList(
+    sortedProductList,
+    filters
+  );
+  const searchedProductList = getSearchedProductList(
+    filteredProductList,
+    search
+  );
+
+  return (
+    <>
+      {isProductSelected ? (
+        <ProductPage />
+      ) : (
+        <div className={styles.productWrapper}>

```

Comment:
Wrap line no 31 to 50 in a component or use a router.

Reply:
Fixed. Used router.